test/unit/UnitTestActions.ts

```diff
@@ -95,4 +98,154 @@ describe('Actions Helpers Unit Tests', () => {
+      const receipt = {
+        logs: [
+          {
+            topics: [ABORTED_EVENT],
+            data: '0x0000000000000000000000000000000000000000000000000000000000000001'
+          }
+        ]
+      };
+
+      const executeStatus = getAbortedExecuteStatus(receipt);
+
+      assert.equal(ExecuteStatus.ABORTED_ALREADY_CALLED, executeStatus);
+    });
+
+    it('returns ExecuteStatus.ABORTED_BEFORE_CALL_WINDOW when AbortReason.BeforeCallWindow', () => {
+      const receipt = {
+        logs: [
+          {
+            topics: [ABORTED_EVENT],
+            data: '0x0000000000000000000000000000000000000000000000000000000000000002'
+          }
+        ]
+      };
+
+      const executeStatus = getAbortedExecuteStatus(receipt);
+
+      assert.equal(ExecuteStatus.ABORTED_BEFORE_CALL_WINDOW, executeStatus);
+    });
+
+    it('returns ExecuteStatus.ABORTED_AFTER_CALL_WINDOW when AbortReason.AfterCallWindow', () => {
+      const receipt = {
+        logs: [
+          {
+            topics: [ABORTED_EVENT],
+            data: '0x0000000000000000000000000000000000000000000000000000000000000003'
+          }
+        ]
+      };
+
+      const executeStatus = getAbortedExecuteStatus(receipt);
+
+      assert.equal(ExecuteStatus.ABORTED_AFTER_CALL_WINDOW, executeStatus);
+    });
+
+    it('returns ExecuteStatus.ABORTED_RESERVED_FOR_CLAIMER when AbortReason.ReservedForClaimer', () => {
+      const receipt = {
+        logs: [
+          {
+            topics: [ABORTED_EVENT],
+            data: '0x0000000000000000000000000000000000000000000000000000000000000004'
+          }
+        ]
+      };
+
+      const executeStatus = getAbortedExecuteStatus(receipt);
+
+      assert.equal(ExecuteStatus.ABORTED_RESERVED_FOR_CLAIMER, executeStatus);
+    });
+
+    it('returns ExecuteStatus.ABORTED_INSUFFICIENT_GAS when AbortReason.InsufficientGas', () => {
+      const receipt = {
+        logs: [
+          {
+            topics: [ABORTED_EVENT],
+            data: '0x0000000000000000000000000000000000000000000000000000000000000005'
+          }
+        ]
+      };
+
+      const executeStatus = getAbortedExecuteStatus(receipt);
+
+      assert.equal(ExecuteStatus.ABORTED_INSUFFICIENT_GAS, executeStatus);
+    });
+
+    it('returns ExecuteStatus.ABORTED_TOO_LOW_GAS_PRICE when AbortReason.TooLowGasPrice', () => {
+      const receipt = {
+        logs: [
+          {
+            topics: [ABORTED_EVENT],
+            data: '0x0000000000000000000000000000000000000000000000000000000000000006'
+          }
+        ]
+      };
+
+      const executeStatus = getAbortedExecuteStatus(receipt);
+
+      assert.equal(ExecuteStatus.ABORTED_TOO_LOW_GAS_PRICE, executeStatus);
+    });
+
+    it('returns ExecuteStatus.ABORTED_UNKNOWN when unknown reason appeared', () => {
+      const receipt = {
+        logs: [
+          {
+            topics: [ABORTED_EVENT],
+            data: '0x0000000000000000000000000000000000000000000000000000000000000008'
```

Comment:
Did you mean to skip `0x0000000000000000000000000000000000000000000000000000000000000007` in the tests? It threw me off for a second.0x0000000000000000000000000000000000000000000000000000000000000008

Reply:
The AbortReason are from 0 to 6, anyway above is unknown from client code pov.